indexing-hadoop/src/main/java/io/druid/indexer/hadoop/HadoopWritableInputRow.java

```diff
@@ -0,0 +1,10 @@
+package io.druid.indexer.hadoop;
+
+import io.druid.data.input.InputRow;
+import org.apache.hadoop.io.Writable;
+
+/**
+ */
+public interface HadoopWritableInputRow extends InputRow, Writable
```

Comment:
Hrm, or, as I think about it more, maybe we do until we start applying the aggregations...  I guess that if we were to actually extract out the dimension tuple as the key and use aggregators to pull out the values, we wouldn't need this to be writable, if we aren't doing that, then yeah, we need it to be writable.  I wonder which is simpler.